toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -306,6 +335,18 @@ func createChroot(workerTar, buildDir, outDir, specsDir string) (chroot *safechr
 		safechroot.NewMountPoint(specsDir, specsMountPoint, "", safechroot.BindMountPointFlags, ""),
 	}
 
+	// Adding the .azure mount ensures the chroot environment can access CLI credentials
+	if useAzureCliAuth {
+		homeDir, homeErr := os.UserHomeDir()
+		if homeErr == nil {
+			azureDir := filepath.Join(homeDir, ".azure")
+			extraMountPoints = append(extraMountPoints, safechroot.NewMountPoint(azureDir, azureMountPoint, "", safechroot.BindMountPointFlags, ""))
+		} else {
+			err = fmt.Errorf("Could not determine user home directory for .azure mount: %v", homeErr)
+			return
+		}
```

Comment:
Could you extract this logic into a separate function as well? Similar to what you did with `installAzureCliPackage`.